Composer/packages/client/src/utils/luUtil.ts

```diff
@@ -20,6 +20,12 @@ export interface LuIntent {
 
 const NEWLINE = '\n';
 
+function isValid(diagnostics: any[]) {
+  return diagnostics.every(item => {
+    item.Severity !== 'ERROR';
```

Comment:
You have defined a `DiagnosticSeverity` constant under `notification/types`.
Use that constant here.

Reply:
not the same cannot use it here